APolitcalApp/src/views/IconTestView.vue

```diff
@@ -0,0 +1,23 @@
+<script lang="ts">
+import IconBase from "../components/icons/IconBase.vue";
+import IconApprove from "../components/icons/IconApprove.vue";
+
+export default{
```

Comment:
This looks like options api, make sure you are using the composition api instead. Basically Vue 2 vs Vue 3